README.md

```diff
@@ -1,54 +1,53 @@
-# Puppet
+# OpenVox
 
-![RSpec tests](https://github.com/puppetlabs/puppet/workflows/RSpec%20tests/badge.svg)
-[![Gem Version](https://badge.fury.io/rb/puppet.svg)](https://badge.fury.io/rb/puppet)
-[![Inline docs](https://inch-ci.org/github/puppetlabs/puppet.svg)](https://inch-ci.org/github/puppetlabs/puppet)
-
-Puppet, an automated administrative engine for your Linux, Unix, and Windows systems, performs
+OpenVox is a community implementation of Puppet, an automated administrative engine for your Linux, Unix, and Windows systems, performs
 administrative tasks (such as adding users, installing packages, and updating server
 configurations) based on a centralized specification.
```

Comment:
```suggestion
OpenVox is a community implementation of Puppet, an automated administrative engine for your Linux, Unix, and Windows systems, designed to perform
administrative tasks (such as adding users, installing packages, and updating server
configurations) based on a centralized specification.
```

Generally an optional explanation clause (the description of puppet) here should be able to be removed and still have the sentence that it generates make sense in language, so I am proposing that we slightly tweak the exit of that clause to make it parse better.